pom.xml

```diff
@@ -197,9 +229,11 @@
             <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- used by GraphitePickleWriter -->
```

Comment:
done , add a comment on it to be more precise 
I currently do not use Graphite ( I use Console Writter or ... a an incomming Custom Outputwriter) so i do not need this dependency